testing/test_monkeypatch.py

```diff
@@ -297,4 +321,10 @@ def test_issue156_undo_staticmethod(Sample):
     monkeypatch.undo()
     assert Sample.hello()
 
-
+def test_issue1338_name_resolving():
+    pytest.importorskip('requests')
```

Comment:
Do we want that dependency, or is a full mock of that specific testcase preferrable?